runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -451,10 +452,10 @@ else if (playerRegionID == KBD_NMZ_REGION)
 					}
 				}
 			}
-			else if (playerRegionID == ZILYANA_REGION || playerRegionID == GRAARDOR_REGION ||
+			else if (playerRegionID == ZILYANA_REGION || playerRegionID == GRAARDOR_REGION || playerRegionID == TEMPOROSS_REGION ||
 				playerRegionID == KRIL_TSUTSAROTH_REGION || playerRegionID == KREEARRA_REGION || playerRegionID == NIGHTMARE_REGION)
```

Comment:
nit-pick: add this new region check to the end instead of the middle:
```suggestion
			else if (playerRegionID == ZILYANA_REGION || playerRegionID == GRAARDOR_REGION ||
				playerRegionID == KRIL_TSUTSAROTH_REGION || playerRegionID == KREEARRA_REGION ||
				playerRegionID == NIGHTMARE_REGION ||  playerRegionID == TEMPOROSS_REGION)
```